src/pages/Subscriptions/SubscriptionBusiness.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import ButtonEstimate from '../../components/Button/ButtonEstimate'
+import ButtonSubscribe from '../../components/Button/ButtonSubscribe'
+import PricingDetailsModal from '../../components/PricingDetailsModal/PricingDetailsModal'
+import PricingDetails from '../../components/PricingDetailsModal/PricingDetails'
+import { useTranslation } from 'react-i18next'
+import { MainView } from './style'
+
+const listPackage = [
+    'up-to-50',
+    'encrypted-calls-EU',
+    'sharescreen"',
+    'invitation-SMS-email-notifs',
+    'group-contact',
+    'reminder-SMS-email',
+    'API-instantvisio',
+]
```

Comment:
pls consider removing code duplication here. This is already declared in `Pricing.tsx`. You may create a `pricing-constants.ts` file for instance